shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -561,6 +568,11 @@ - (void)setupChannels {
       binaryMessenger:self.binaryMessenger
                 codec:[FlutterJSONMethodCodec sharedInstance]]);
 
+  _spellCheckChannel.reset([[FlutterMethodChannel alloc]
+         initWithName:@"flutter/spellcheck"
+      binaryMessenger:self.binaryMessenger
+                codec:[FlutterJSONMethodCodec sharedInstance]]);
```

Comment:
drive by: new codecs should avoid using the JSON codec.

Reply:
What codec should be used?

Reply:
FlutterStandardMethodCodec